jwst/assign_wcs/miri.py

```diff
@@ -69,9 +69,9 @@ def imaging(input_model, reference_files):
     try:
         bb = distortion.bounding_box
     except NotImplementedError:
-        shape = input_model.data.shape
+        shape = 1032,1024
```

Comment:
@drlaw1558 This change here made me think about how `bounding_box` works with subarrays. Hard coding shape to the full frame size means that with subarrays the bounding_box will be larger than the image. As an example, for a subarray with xstart=1000, xsize=30, the image `x` coordinates are from 0 to 30. Input of x= 100 is within the bounding_box, however in full frame coordinates `x_full=1032` and so is outside the full frame.

If we keep the bounding_box to the image shape, this problem is solved. However, in this case, for a subarray and for inputs outside the subarray size, the inputs will be considered outside the bounding_box when in fact the distortion is defined there because it's defined over the full frame. But may be this is not a problem and we want to keep the bounding_box to the image size?